### ISSUE TITLE
LitEmoji error when saving empty HTML field

### ISSUE THREAD

**Issue by jmauzyk** [verbb/formie#1402]
### Describe the bug

Occasionally will use empty HTML fields to organize the form layout in a particular way. Getting the following error when trying to save a form with an empty HTML field:

```
2023-04-19 16:26:12 [ERROR] Unable to save form “contact”: LitEmoji\LitEmoji::unicodeToShortcode(): Argument #1 ($content) must be of type string, null given, called in /home/parktool/craft-4/vendor/verbb/formie/src/fields/formfields/Html.php on line 119
```

### Steps to reproduce

1. Add an HTML field to a form and leave the content empty.
2. Click Save.

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

4.4.7.1

### Plugin version

2.0.27.1

### Multi-site?

Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.0.27.1"`

**Comment by engram-design** [comment]
Fixed in [2.0.28](https://github.com/verbb/formie/releases/tag/2.0.28)
